pkg/network/tracer/tracer_windows.go

```diff
@@ -52,13 +57,85 @@ type Tracer struct {
 	destExcludes   []*network.ConnectionFilter
 }
 
+func isClosedSourceAllowed() bool {
+	ddRegKey := `SOFTWARE\DataDog\Datadog Agent`
+	regKey, err := registry.OpenKey(registry.LOCAL_MACHINE, `SOFTWARE\DataDog\Datadog Agent`, registry.QUERY_VALUE)
+	if err != nil {
+		log.Warnf("unable to open registry key %s: %v", ddRegKey, err)
+		return false
+	}
+	defer regKey.Close()
+
+	ddRegName := "AllowClosedSource"
+	val, _, err := regKey.GetIntegerValue("AllowClosedSource")
+	if err != nil {
+		log.Warnf("unable to get value for %s: %v", ddRegName, err)
+		return false
+	}
+
+	if val == 1 {
+		log.Infof("%s is set; closed-source software allowed", ddRegName)
+		return true
+	} else if val == 0 {
+		log.Infof("%s is not set; closed-source software not allowed", ddRegName)
```

Comment:
The `infof` doesn't match the case (which also speaks to a larger issue in a sec.)
The `val == 0` case is where the value was explicitly zero.  If the value doesn't exist (not set), it returns `ErrNotExist`.

So in the case where it's not set, it will actually catch the "unable to get value" error above on line 72.

Which begs the larger question, which I know was discussed but I can't remember:  Is it a two-state or 3 state?